hakyll.cabal

```
@@ -246,7 +248,7 @@ Library
     Other-Modules:
       Hakyll.Web.Pandoc.Binary
     Build-Depends:
-      pandoc >= 2.11 && < 2.20
+      pandoc >= 2.11 && < 3.1
```

I suggest this is changed to a more restrictive clause similar to the ones for `aeson` and `text`:

```suggestion
      pandoc >= 2.11 && < 2.19.2 || >= 3.0 && < 3.1
```

Ditto in two places below.